src/api/specimen/GetRecentSpecimens.ts

```diff
@@ -11,6 +11,7 @@ import { Specimen, JSONResultArray, Dict } from 'global/Types';
 const GetRecentSpecimens = async (pageSize: number, pageNumber?: number) => {
     let recentSpecimens = [] as Specimen[];
     let links: Dict = {};
+    let totalRecords: number = 25;
```

Comment:
Well, it remains the same as the pageSize for now because there is now totalRecords returned from the generic /specimens endpoint. However, now I think about it, when a page contains less results this will be false. Otherwise, what should we show here? Because in theory, it is possible to almost click on forever since it fetches from the total amount of specimens in DiSSCo.